app/views/events/new.html.erb

```diff
@@ -2,4 +2,5 @@
   <h1>Add a New Event</h1>
 
   <%= render 'form'%>
+  <%= link_to 'Back', events_path, class: "btn btn-primary"%>
```

Comment:
Can we give it some margin on top so it doesn't appear stuck to the other button?